numpy/core/defchararray.py

```diff
@@ -551,21 +551,21 @@ def decode(a, encoding=None, errors=None):
 
     See Also
     --------
-    str.decode
+    :py:meth:`bytes.decode`
 
     Notes
     -----
     The type of the result will depend on the encoding specified.
 
     Examples
     --------
-    >>> c = np.array(['aAaAaA', '  aA  ', 'abBABba'])
+    >>> c = np.array([b'\x81\xc1\x81\xc1\x81\xc1', b'@@\x81\xc1@@',
+                      b'\x81\x82\xc2\xc1\xc2\x82\x81'])
```

Comment:
In an interactive Python shell, when a line is continued, the prompt becomes `... `:
```suggestion
...                   b'\x81\x82\xc2\xc1\xc2\x82\x81'])
```